frontend/public/components/_catalog.scss

```diff
@@ -111,5 +111,10 @@ $catalog-item-icon-size-sm: 24px;
     flex: 0 0 190px;
     margin: 0 ($grid-gutter-width / 2) 0 0;
   }
+
+  .blank-slate-pf {
+    background-color: inherit;
+    border: none;
+  }
```

Comment:
Rather than overriding, you can add the Pf class `blank-slate-content-pf` to your empty state component.

Reply:
fixed